guardian/backends.py

```diff
@@ -113,6 +116,63 @@ def has_perm(self, user_obj: Any, perm: str, obj: Optional[Model] = None) -> boo
         check = ObjectPermissionChecker(user_obj)
         return check.has_perm(perm, obj)
 
+    if _supports_async:
+
+        async def ahas_perm(self, user_obj: Any, perm: str, obj: Optional[Model] = None) -> bool:
```

Comment:
The default implementation checks if `perm in await self.get_all_permissions(...)`. Keeping it would be a problem since we currently do not return the full permission name (including the `app_label`) on those methods. 🤔